### DISCUSSION TITLE
Consider adding support to EIP1271 (Standard Signature Validation Method for Contracts)

### DISCUSSION THREAD

**Discussion by cygnusv** [PaulRBerg/prb-proxy#41] [Ideas]
Since the proxy can't sign messages, it would be to consider alternative methods that allow to validate signatures on behalf of the proxy. For this purposes, [EIP1271](https://eips.ethereum.org/EIPS/eip-1271) it's a good place to start

**Comment by PaulRBerg** [comment]
Thanks, this is a great suggestion!

**Comment by cleanunicorn** [comment]
@cygnusv Check out this discussion (plugin support) https://github.com/paulrberg/prb-proxy/discussions/42, since that type of extension will allow for EIP-1271 to be implemented.

**Comment by PaulRBerg** [comment]
As @cleanunicorn has said, PRBProxy now implicitly supports EIP-1271 via the plugin system.
